fades/main.py

```diff
@@ -181,13 +182,11 @@ def go():
     if args.clean_unused_venvs:
         try:
             max_days_to_keep = int(args.clean_unused_venvs)
-            usage_manager.clean_unused_venvs(max_days_to_keep)
-        except Exception:
-            rc = 1
-            logger.debug("CLEAN_UNUSED_VENVS must be an integer.")
+        except ValueError:
+            logger.error("clean_unused_venvs must be an integer.")
             raise
-        finally:
-            sys.exit(rc)
+
+        usage_manager.clean_unused_venvs(max_days_to_keep)
```

Comment:
After this operation, fades must quit (this was handled in the `finally` you removed)